stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  description: "S3 bucket accesskey"
+  type: string
+  label: s3 AccessKey
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.secretkey
+  required: true
+  default: ""
+  description: "S3 bucket secretkey"
+  type: string
+  label: s3 SecretKey
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.regionendpoint
+  default: ""
+  description: "S3 bucket endpoint"
+  type: string
+  label: s3 Bucket Endpoint
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+# oss storage
+- variable: persistence.imageChartStorage.oss.bucket
+  required: true
+  default: ""
+  description: "Aliyun oss bucket name"
+  type: string
+  label: Aliyun oss Bucket Name
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.region
+  required: true
+  default: ""
+  description: "Aliyun oss region name"
+  type: string
+  label: Aliyun oss Bucket Region
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.accesskeyid
+  required: true
+  default: ""
+  description: "Aliyun oss accesskey id"
+  type: string
+  label: Aliyun oss AccessKey ID
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.accesskeysecret
+  required: true
+  default: ""
+  description: "Aliyun oss accesskey secret"
+  type: string
+  label: Aliyun oss AccessKey Secret
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+# swift storage
+- variable: persistence.imageChartStorage.swift.authurl
+  required: true
+  default: "https://storage.myprovider.com/v3/auth"
+  type: string
+  label: Swift Auth URL
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.swift.username
+  required: true
+  default: ""
+  type: string
+  label: Swift Username
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.swift.password
+  required: true
+  default: ""
+  type: string
+  label: Swift Password
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.swift.container
+  required: true
+  default: ""
+  type: string
+  label: Swift Storage Container Name
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+# registry config
+- variable: persistence.persistentVolumeClaim.registry.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
```

Comment:
What is "Default to null"? You mean "defaults"? But it defaults to empty string which is not the same as null...

Reply:
Another thing I inherited from the original Harbor questions.yaml file. I changed the descriptions and labels for all PVCs to make them more user friendly.